baseapp/abci.go

```diff
@@ -106,10 +107,13 @@ func (app *BaseApp) InitChain(req abci.RequestInitChain) (res abci.ResponseInitC
 func (app *BaseApp) Info(req abci.RequestInfo) abci.ResponseInfo {
 	lastCommitID := app.cms.LastCommitID()
 
+	v := &tmproto.VersionParams{}
+	app.paramStore.Get(app.deliverState.ctx, ParamStoreKeyVersionParams, v)
```

Comment:
Does the upgrade module increment the app version in `app.paramStore` when an upgrade happens?